docs/rest-apis/devportal/webhook-subscribers.md

```
@@ -552,3 +550,113 @@ This operation requires <strong>Basic Auth</strong> authentication.
+
+</aside>
+
+<h3 id="list-recent-deliveries-for-a-webhook-subscriber-parameters">Parameters</h3>
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|orgId|path|string|true|none|
+|subscriberId|path|string|true|Webhook subscriber ID (UUID).|
+
+> Example responses
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)